docs/source/en/model_doc/dinov2.md

```diff
@@ -50,31 +63,14 @@ processor = AutoImageProcessor.from_pretrained('facebook/dinov2-base')
 model = AutoModel.from_pretrained('facebook/dinov2-base')
 
 inputs = processor(images=image, return_tensors="pt")
-outputs = model(**inputs)
-last_hidden_states = outputs[0]
-
-# We have to force return_dict=False for tracing
-model.config.return_dict = False
-
-with torch.no_grad():
-    traced_model = torch.jit.trace(model, [inputs.pixel_values])
-    traced_outputs = traced_model(inputs.pixel_values)
-
-print((last_hidden_states - traced_outputs[0]).abs().max())
+image_embedding = model(**inputs)[0][0].mean(dim=0)
 ```
 
-## Resources
-
-A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DINOv2.
-
-- Demo notebooks for DINOv2 can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DINOv2). 🌎
-
-<PipelineTag pipeline="image-classification"/>
+</hfoption>
 
-- [`Dinov2ForImageClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/image_classification.ipynb).
-- See also: [Image classification task guide](../tasks/image_classification)
+## Notes
 
-If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+- [`DINOv2 with Registers`] 
```

Comment:
Add indented code snippet of tracing under here.

```suggestion
- Use [torch.jit.trace](https://pytorch.org/docs/stable/generated/torch.jit.trace.html) to speedup inference. However, it will produce some mismatched elements. The difference between the original and traced model is 1e-4.

   <code>
```